php/Terminus/User.php

```diff
@@ -73,12 +75,21 @@ public function sites($organization=null) {
     return $response['data'];
   }
 
-  public function getId() {
-    return $this->id;
+  public function instruments() {
+    $instruments       = new Instruments(array('user' => $this));
```

Comment:
Really, shouldn't these only be instantiated right before a fetch? Why have a useless collection?